templates/deno/app/entry.server.tsx

```diff
@@ -1,15 +1,15 @@
-import * as React from "react";
-import { renderToString } from "react-dom/server";
-import { RemixServer } from "remix";
-import type { EntryContext } from "remix";
+import  React from "./deps/react.ts";
+import  ReactDOMServer from "./deps/react-dom-server.ts";
```

Comment:
Looks like you have an extra space on these two lines after the `import` keyword